lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -22,6 +22,8 @@ class KubernetesResource
       If you have reason to believe it will succeed, retry the deploy to continue to monitor the rollout.
       MSG
 
+    TIMEOUT_OVERRIDE_ANNOTATION = "kubernetes-deploy.shopify.io/timeout-override-seconds"
```

Comment:
Would it be possible to support more ways to specify the duration like `10s` or `5m`? Or perhaps just make the annotation `kubernetes-deploy.shopify.io/timeout-override` and state that for the time being the only acceptable value is seconds?

I am bringing this up because a timeout of `15m` would be `900s` which isn't as easy for a human to parse.

Reply:
That's fair. I'll update it to accept "s" or "m", and to assume that unqualified values are seconds.

Reply:
Well if you're going to have `m` and `s`, you might as well have `h`...

Reply:
I recommend `ISO8601` format (to be used with `ActiveSupport::Duration::ISO8601Parse`) for time durations. It's what I used in my open PR on resource discovery.